lmdeploy/archs.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Literal, Optional, Union
+
+from transformers import AutoConfig
+
+from .messages import PytorchEngineConfig, TurbomindEngineConfig
+
+_SUPPORTED_ARCHS = dict(
+    # baichuan-7b
+    BaiChuanForCausalLM=dict(pytorch=False, turbomind=True),
+    # baichuan2-7b, baichuan-13b, baichuan2-13b
+    BaichuanForCausalLM=dict(pytorch=True, turbomind=True),
+    # chatglm2-6b, chatglm3-6b
+    ChatGLMModel=dict(pytorch=True, turbomind=False),
+    # deepseek-moe
+    DeepseekForCausalLM=dict(pytorch=True, turbomind=False),
+    # falcon-7b
+    FalconForCausalLM=dict(pytorch=True, turbomind=False),
+    # gemma-7b
+    GemmaForCausalLM=dict(pytorch=False, turbomind=False),
```

Comment:
need to update here after pr https://github.com/InternLM/lmdeploy/pull/1184

Reply:
PR #1184 has been merged.

Reply:
ok